Java/README.md

```diff
@@ -21,4 +21,8 @@ Add `JAVA_HOME` to your environment variables by adding the line below to your `
 
     export JAVA_HOME="`/usr/libexec/java_home -v 1.8`"
 
+Are you Java9 user? Use below instead of above.
+
+    export JAVA_HOME="`/usr/libexec/java_home -v 9`"
```

Comment:
Are backticks actually needed?

Reply:
They are indeed needed, I just tried it. And we have them for the Java 8 example above.